styles.css

```diff
@@ -508,4 +510,44 @@
   font-size: 18px;
   line-height: 28px;
   color: #d9dbe1;
-}
\ No newline at end of file
+}
+
+@media (max-width: 1030px) {
```

Comment:
- [ ] Always try to apply media query for standard breakpoints

Reply:
Ok note taken. Actually in my opinion I don't think this PR needs to be merged because I rewrote all the media queries in the second PR (sow/resp-hero-section)